apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/FinalDecisionDialog.tsx

```diff
@@ -0,0 +1,339 @@
+                  },
+                ]}
+              />
+              {finalAssessmentDecisionValue === "disqualified" && (
+                <RadioGroup
+                  idPrefix="disqualifiedDecision"
+                  name="disqualifiedDecision"
+                  legend={intl.formatMessage({
+                    defaultMessage: "Disqualified decision",
+                    description: "Disqualified decision input",
+                    id: "Ed+xy1",
+                  })}
+                  rules={{
+                    required: intl.formatMessage(errorMessages.required),
+                  }}
+                  items={[
+                    {
+                      value: "application",
+                      label: intl.formatMessage({
+                        defaultMessage: "Screened out on application",
+                        description: "Screened out on application option",
+                        id: "YueN6y",
+                      }),
+                    },
+                    {
+                      value: "assessment",
+                      label: intl.formatMessage({
+                        defaultMessage: "Unsuccessful during assessment",
+                        description: "Unsuccessful during assessment option",
+                        id: "wrqRvV",
+                      }),
+                    },
+                  ]}
+                  data-h2-margin-top="base(x1)"
+                />
+              )}
+              {finalAssessmentDecisionValue === "qualified" && (
+                <>
+                  <Heading
+                    data-h2-font-weight="base(700)"
+                    data-h2-margin-bottom="base(x.5)"
+                    data-h2-font-size="base(normal)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Select an expiry date",
+                      id: "3MboNR",
+                      description:
+                        "Label for date selection input for expiry date",
+                    })}
+                  </Heading>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "This is the amount of time this candidate will be considered for placement based on the results of this process. The usual amount of time is two years.",
+                      id: "DXjJyD",
+                      description:
+                        "Text describing expiry dates for candidates, what it is for and a recommendation",
+                    })}
+                  </p>
+                  <DateInput
+                    id="expiryDate"
+                    name="expiryDate"
+                    legend={intl.formatMessage({
+                      defaultMessage: "Select an expiry date",
+                      id: "3MboNR",
+                      description:
+                        "Label for date selection input for expiry date",
+                    })}
```

Comment:
Currently varying a bit from the Figma target, as the date input also needs a label/legend

Current

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/e6ee3834-0bb9-4bff-b2fd-3097dd95c007)

Reply:
Hmm, I don't think this works for design but it could be avoided if we use the `context` prop for the description of the field (its true purpose). But that would move the text below the input 🤔

Reply:
@Jerryescandon 
Thoughts on the above and Eric's suggestion?

Reply:
🤔 Yeah I think that would work well.. I've updated the figma.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/62c4c350-4548-4007-a899-4b6838cfc64e)

Reply:
Thank you very much!

Reply:
Context used in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8978/commits/ac39b7a093976cd12675c1ef977ed9ed0f871074